app/assets/javascripts/experiments/table.js

```diff
@@ -55,6 +55,41 @@ var ExperimnetTable = {
         </div>`;
       $(`<div class="table-row" data-urls='${JSON.stringify(data.urls)}' data-id="${id}">${row}</div>`)
         .appendTo(`${this.table} .table-body`);
+      this.initDueDatePicker(`#calendarDueDate${data.columns.filter(k => k.column_type === 'id')[0].data.id}`);
```

Comment:
Can you move it to separate action and just call this action here.
And you have ID available here with just `id`, check line 56